doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -441,6 +441,109 @@ Here are examples:
    end
    where "n + m" := (plus n m).
 
+Enabling and disabling notations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. cmd:: {| Enable | Disable } Notation {? {| @string | @qualid } } {? := @one_term } {? ( {+, @toggle_notation_flag } ) } {? {| : no scope | : @ident } }
+   :name: Disable Notation
+
+   .. insertprodn toggle_notation_flag toggle_notation_flag toggle_notation_flag
+
+   .. prodn::
+      toggle_notation_flag ::= all
+      | only parsing
+      | only printing
+      | in custom @ident
+      | in constr
+
+   Enables or disables notations previously defined with
+   :cmd:`Notation` or :cmd:`Notation (abbreviation)`. At least one of
+   :token:`string`, :token:`qualid`, :token:`one_term` or scope must be
+   provided. If more than one is provided, the notations enabled or
+   disabled are the ones meeting the conjunction of these constraints.
+
+   This command supports the :attr:`local` and :attr:`global`
+   attributes.
+
+   .. note::
+      This command has no effect on parser modifications made with
+      :cmd:`Reserved Notation`.
+
+   :n:`@string`
+      Notations to enable or disable. :n:`@string` can be a single
+      token in the notation such as "`->`" or a pattern that matches
+      the notation. See :ref:`locating-notations`.
+
+   :n:`@qualid`
+      :ref:`Abbreviation <Abbreviations>` to enable or disable.
+
+   :n:`{? := @one_term }`
+      Enable or disable notations involving :token:`one_term`.
+
+   :n:`{? {| : no scope | : @ident } }`
+      If given, only notations in scope :token:`ident` are affected (or
+      :term:`lonely notations <lonely notation>` for ``no scope``).
+
+   :n:`all`
+      Enable or disable all notations meeting the given constraints,
+      even if multiple ones. When not provided, there must be a single
+      notation meeting the constraints.
+
+   :n:`only parsing`
+      The notation is enabled or disabled only for parsing.
+
+   :n:`only printing`
+      The notation is enabled or disabled only for printing.
+
+   :n:`in custom @ident`
+      Enable or disable notations in given :ref:`custom entry
+      <custom-entries>`.
+
+   :n:`in constr`
+      Enable or disable notations in ``constr``. See
+      :ref:`custom entries <custom-entries>`.
```

Comment:
Well, constr here is a particular case of custom entry, hence the explicit reference to custom entries where this is explained (after a few paragraphs, maybe this would require a more precise reference).